src/docker-osx-dev

```diff
@@ -627,16 +622,14 @@ function brew_install {
   local readonly command_name="$3"
   local readonly use_cask="$4"
 
-  local brew_command="brew"
-  if [[ "$use_cask" = true ]]; then
-    brew_command="brew cask"
-  fi
+  if ! type "$command_name" > /dev/null 2>&1 ; then
+    check_homebrew $readable_name
+
+    local brew_command="brew"
+    if [[ "$use_cask" = true ]]; then
+      brew_command="brew cask"
+    fi
 
-  if eval "$brew_command list $package_name" > /dev/null 2>&1 ; then
```

Comment:
Just ran this locally, and I think you lost this check. As a result, it tried to install Brew Cask, which I already had installed, and exited with an error.